src/skeptic/lib.rs

```diff
@@ -60,7 +60,7 @@ pub fn markdown_files_of_directory(dir: &str) -> Vec<String> {
 /// use skeptic::generate_doc_tests;
 /// 
 /// fn main() {
-///     generate_doc_tests(&["README.md"]);
+///     generate_doc_tests(&["README.md".into()]);
```

Comment:
Please remove this change. Compiler will not be able to infer the type if you use `"README.md".into()` here but leaving the bare `"README.md"` will do the trick. 
please run `cargo test` in both the root `rust-skeptic` dir as well as as in `rust-skeptic/src/skeptic`
The firs one runs integration tests and the second one unit tests (unfortunately these are not connected at the moment)

Reply:
makes sense.  Thanks for being patient with me while I fumble around Rust. Final (hopefully) change submitted.